src/NServiceBus.AcceptanceTests/.editorconfig

```diff
@@ -4,4 +4,7 @@
 dotnet_diagnostic.CA1711.severity = none
 
 # Justification: Test project
-dotnet_diagnostic.CA2007.severity = none
\ No newline at end of file
+dotnet_diagnostic.CA2007.severity = none
+
+# IDE0028: Simplify collection initialization
```

Comment:
**Nitpick**: Can you please update this comment to have the same format as the others, meaning: "Justification:..."

Reply:
Nope, once the code gets to be a support branch, there's not a lot of gain in going beyond what the Visual Studio tooling creates when you change the warning level.

Reply:
No, I meant to change the text on the comment to have the same format as other comments on the editorconfig files.
On this file you put {rule}: {name of the rule}, but on others it is "Justification: {reason why it is different}"